packages/manager/src/features/linodes/LinodesCreate/LinodeCreate.tsx

```diff
@@ -733,20 +741,41 @@ export class LinodeCreate extends React.PureComponent<
                 />
               ) : null}
             </div>
-            <Button
-              data-qa-deploy-linode
-              buttonType="primary"
-              onClick={this.createLinode}
-              loading={formIsSubmitting}
-              className={classes.createButton}
-              disabled={
-                formIsSubmitting ||
-                userCannotCreateLinode ||
-                (showAgreement && !signedAgreement)
-              }
-            >
-              Create Linode
-            </Button>
+            <div>
+              <Button
+                data-qa-api-cli-linode
+                buttonType="outlined"
+                onClick={this.toggleApiAwarenessModal.bind(null, true)}
```

Comment:
Throughout the repo we prefer the following

```suggestion
                onClick={() => this.toggleApiAwarenessModal(true)}
```

As much as I dislike arrow functions, I find this more readable.